team.md

```diff
@@ -7,6 +7,17 @@ permalink: /team/
 
 ## Our Team
 
-Find out more about the individuals who make up Astral Dynamics.
+Astral Dynamics is a diverse multi-disciplinary team. We have often worked with guest members on voyages,
+and taken on interns for periods up to 1-year. Below we've listed the core team members who are full-time
+crew. 
+
+In October 2015, we decided to spend a weekend having a go at an innovation 
+competition on quantified self. 
+One of the requirements was to make a video, so using only a phone camera and with only 24hrs to film and 
+edit it we made this video. We thought we'd share it here by way of introducing out team. 
```

Comment:
Cut this cruft. Just start with the video and have a short description explaining that it was a promo video made for a project we worked on recently. The pre-emptive back pedaling on phone camera and 24h just sounds sloppy.

If we think it's high enough quality to live here, then we don't apologise for it. If we don't, then simply we don't use it.